.devcontainer/Dockerfile

```diff
@@ -5,8 +5,10 @@ ARG VARIANT="1.17-bullseye"
 FROM mcr.microsoft.com/vscode/devcontainers/go:0-${VARIANT}
 
 # [Choice] Node.js version: none, lts/*, 16, 14, 12, 10
-ARG NODE_VERSION="none"
-RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+#ARG NODE_VERSION="none"
+#RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+
+RUN rm /etc/apt/sources.list.d/yarn.list
```

Comment:
The gpg key for yarn expired. I tried multiple commands to update the apt gpg keys, but none of them worked. We don't need yarn in this repo so I just removed it for now.